scripts/init-data.sql

```diff
@@ -0,0 +1,101 @@
+create database giftcertificatedatabase;
```

Comment:
Please remove the file if it's not used. init.sql is enough

Reply:
Removed